api/v1alpha1/directoryservice_types.go

```diff
@@ -205,6 +209,17 @@ type DirectoryServiceList struct {
 	Items           []DirectoryService `json:"items"`
 }
 
+// MultiRegion enables MCS and configures region suffixes for multiple multi-region solutions
+type MultiRegion struct {
+	// +kubebuilder:default:=false
```

Comment:
Lee - I wonder if we should rename all the MultiRegion to MultiCluster?  Which aligns with the K8S terminology and MCS
You could have MCS in the same region..

Reply:
Good point, I was toying with both. I'll change to MultiCluster